MyActiveRunsPanel & RecentlyCompletedRunsPanel scroll problem fixed

TimSPb89 opened the pull request:
This PR fixes an issue with the scroll on the MyActiveRunsPanel and the RecentlyCompletedRunsPanel panels.

TimSPb89 committed:
MyActiveRunsPanel & RecentlyCompletedRunsPanel scroll problem fixed

sidoruka commented:
This PR brakes the Dashboard's Active Runs view. Reverted from `develop`
@TimSPb89 please fix and let @rodichenko review

TimSPb89 commented:
> This PR brakes the Dashboard's Active Runs view. Reverted from `develop`
> @TimSPb89 please fix and let @rodichenko review

Fixed by #891 
@rodichenko please, take a look when you have time
